conda/recipes/ucxx/conda_build_config.yaml

```diff
@@ -10,6 +10,9 @@ sysroot_linux_64:
 cmake:
   - ">=3.23.1,!=3.25.0"
 
+rmm_version:
+  - "23.04.*"
```

Comment:
Does this get updated by the `ci/release/update-version.sh` script? If not, should it be?